api/src/paths/telemetry/vendor/deployments.ts

```diff
@@ -5,7 +5,7 @@ import { getBctwUser } from '../../../services/bctw-service/bctw-service';
 import { BctwTelemetryService } from '../../../services/bctw-service/bctw-telemetry-service';
 import { getLogger } from '../../../utils/logger';
 
-const defaultLog = getLogger('paths/telemetry/manual');
+const defaultLog = getLogger('paths/telemetry/vendor/deployments');
```

Comment:
Should this endpoint end with ```telemetry```? like vendor/deployments/telemetry? I would assume that fetching deployments would omit telemetry points

Reply:
Most of the related code will need to be updated or removed, etc, as part of the larger work needed to update the Manage Telemetry page.